distribution/build.gradle

```diff
@@ -73,9 +70,9 @@ publishing {
             }
         }
 
-        subjectExtension(MavenPublication) {
-            groupId = 'org.jetbrains.spek'
-            artifactId = 'spek-subject-extension'
+        extensionSubject(MavenPublication) {
+            groupId = 'org.spekframework.spek2'
```

Comment:
Latest change extracted `groupId` and `version`, it took advantage of the `maven-publish`'s default values.